lib/Mojo/Util.pm

```diff
@@ -88,6 +88,33 @@ monkey_patch(__PACKAGE__, 'sha1_sum',      \&sha1_hex);
 monkey_patch(__PACKAGE__, 'steady_time',
   MONOTONIC ? sub () { Time::HiRes::clock_gettime(Time::HiRes::CLOCK_MONOTONIC()) } : \&Time::HiRes::time);
 
+# Building random bytes function
+monkey_patch(__PACKAGE__, 'urandom_bytes', $^O eq 'MSWin32' ? _build_urandom_bytes_win32() : _build_urandom_bytes());
```

Comment:
This seems over-optimized, this can just be an if condition within the function.

Reply:
I'd additionally recommend a constant something like this so such a condition can be inlined by the parser and it can be easily managed later:

```
use constant WIN32_URANDOM => $^O eq 'MSWin32';
```

Reply:
This is fixed